app/pods/components/player/player-section-list-item/template.hbs

```diff
@@ -6,11 +6,11 @@
           {{section.name}}
 
           </h6>
-          {{#if (not section.premium)}}
-            <div class="pill bg-gradient-orange white card-md normal ml-1">
-              Free
-            </div>
-          {{/if}}
+            {{#if (and (not section.premium ) (not premiumStudent) )}}
```

Comment:
combine these to yet another computed property -> `showFreeTag`

Keep your templates as declarative as possible, reading through them should be idiomatic